lib/Page.js

```diff
@@ -136,6 +137,14 @@ Page.prototype.removeFrontMatter = function (includedPage) {
   return $.html();
 };
 
+Page.prototype.collectHeadings = function (includedPage) {
+  const $ = cheerio.load(includedPage);
+  $('h1,h2,h3').map((i, heading) => {
```

Comment:
This should use `forEach` rather than `map` so that we can avoid the `return 0;`

Reply:
Sorry, the correct name should be `each()` for this case.